Tutorial Proposals/GPIO Series/Digital IO/1.1 Blink an LED.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 1.1 Blink an LED
+---
+
+## Introduction
+
+This tutorial is designed for beginners who are interested in learning how to interact with the
+physical world using the Raspberry Pi. The reader learns how to wire an LED to the Raspberry Pi
+and control it to blink at regular intervals. This project is a foundational exercise in
+understanding General Purpose Input/Output (GPIO) pins.
+
+## Prerequisites
+
+- Basic understanding of electronics.
+- Raspberry Pi setup with SplashKit installed.
```

Comment:
Might be useful to  put a link here with how to do this E.G.

Raspberry Pi setup with SplashKit [installed](https://splashkit.io/installation/linux/step-1/).